FUNDING.yml

```diff
@@ -0,0 +1,19 @@
+# If used uncomment and update the following:
+
+# ####################################################################################
+# # Used to define funding options for your project or organization on GitHub. 
+# # It's part of GitHub's Sponsors feature, 
+# # which allows open source developers and organizations to receive financial support for their work.
+# ####################################################################################
+
+# # These are supported funding model platforms
+# github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+# patreon: # Replace with a single Patreon username
+# open_collective: # Replace with a single Open Collective username
+# ko_fi: # Replace with a single Ko-fi username
+# tidelift: # Replace with a single Tidelift platform-name/package-name e.g., npm/babel
+# community_bridge: # Replace with a single Community Bridge project-name e.g., cloud-foundry
+# liberapay: # Replace with a single Liberapay username
+# issuehunt: # Replace with a single IssueHunt username
+# otechie: # Replace with a single Otechie username
+# custom: # Replace with up to 4 custom sponsorship URLs e.g., ['link1', 'link2']
```

Comment:
both for consistency and good practice, all files should have an empty line at the end unless its format won't allow it

```suggestion
# custom: # Replace with up to 4 custom sponsorship URLs e.g., ['link1', 'link2']

```